pkg/ccl/streamingccl/streamingest/stream_replication_e2e_test.go

```diff
@@ -778,7 +809,8 @@ func TestTenantStreamingUnavailableStreamAddress(t *testing.T) {
 
 	// Once srcCluster.Server(0) is shut down queries must be ran against a different server
 	alternateSrcSysSQL := sqlutils.MakeSQLRunner(c.srcCluster.ServerConn(1))
-	_, alternateSrcTenantConn := serverutils.StartTenant(t, c.srcCluster.Server(1), base.TestTenantArgs{TenantID: c.args.srcTenantID, DisableCreateTenant: true, SkipTenantCheck: true})
+	_, alternateSrcTenantConn := serverutils.StartTenant(t, c.srcCluster.Server(1),
+		base.TestTenantArgs{TenantID: c.args.srcTenantID, DisableCreateTenant: true, SkipTenantCheck: true})
```

Comment:
This part we can just leave `SkipTenantCheck` being unset, since the source tenant should exist and should be active and we don't care why this fails if that isn't the case.  The purpose of the other place with `SkipTenantCheck` was to try to connect to an inactive destination tenant and verify that even without a testserver's TenantCheck it still fails.